dev/devicelab/lib/framework/apk_utils.dart

```diff
@@ -83,6 +83,83 @@ bool hasMultipleOccurrences(String text, Pattern pattern) {
   return text.indexOf(pattern) != text.lastIndexOf(pattern);
 }
 
+
+/// Utility class to analyze the content inside an APK.
+class ApkExtractor {
```

Comment:
This class should have some unit tests if possible

Reply:
This is a utility for the devicelab tests, so it's only used for testing code.

Reply:
@jonahwilliams how should this be unit tested? It looks like there are some tests of devicelab code under dev/devicelab/test. Is there a DI setup as in the flutter_tool?